balrogscript/tests/conftest.py

```diff
@@ -28,61 +29,73 @@ def release_manifest():
+                    "submit-toplevel": "balrogscript/data/balrog_submit-toplevel_schema.json",
+                    "schedule": "balrogscript/data/balrog_schedule_schema.json",
                 },
-                "release": {
-                    "api_root": "BALROG_API_ROOT",
-                    "auth0_domain": "AUTH0_DOMAIN",
-                    "auth0_client_id": "AUTH0_CLIENT_ID",
-                    "auth0_client_secret": "AUTH0_CLIENT_SECRET",
-                    "auth0_audience": "AUTH0_AUDIENCE",
-                    "allowed_channels": ["release", "release-localtest", "release-cdntest"],
+                "dummy": False,
+                "api_root": "BALROG_API_ROOT",
+                "taskcluster_scope_prefix": "project:releng:balrog:",
+                "server_config": {
+                    "nightly": {
+                        "api_root": "BALROG_API_ROOT",
+                        "auth0_domain": "AUTH0_DOMAIN",
+                        "auth0_client_id": "AUTH0_CLIENT_ID",
+                        "auth0_client_secret": "AUTH0_CLIENT_SECRET",
+                        "auth0_audience": "AUTH0_AUDIENCE",
+                        "allowed_channels": ["nightly"],
+                    },
+                    "release": {
+                        "api_root": "BALROG_API_ROOT",
+                        "auth0_domain": "AUTH0_DOMAIN",
+                        "auth0_client_id": "AUTH0_CLIENT_ID",
+                        "auth0_client_secret": "AUTH0_CLIENT_SECRET",
+                        "auth0_audience": "AUTH0_AUDIENCE",
+                        "allowed_channels": ["release", "release-localtest", "release-cdntest"],
+                    },
+                    "dep": {
+                        "api_root": "BALROG_API_ROOT",
+                        "auth0_domain": "AUTH0_DOMAIN",
+                        "auth0_client_id": "AUTH0_CLIENT_ID",
+                        "auth0_client_secret": "AUTH0_CLIENT_SECRET",
+                        "auth0_audience": "AUTH0_AUDIENCE",
+                        "allowed_channels": ["nightly", "release", "beta", "etc"],
+                    },
                 },
-                "dep": {
-                    "api_root": "BALROG_API_ROOT",
-                    "auth0_domain": "AUTH0_DOMAIN",
-                    "auth0_client_id": "AUTH0_CLIENT_ID",
-                    "auth0_client_secret": "AUTH0_CLIENT_SECRET",
-                    "auth0_audience": "AUTH0_AUDIENCE",
-                    "allowed_channels": ["nightly", "release", "beta", "etc"],
-                },
-            },
-            "disable_certs": False,
-            "verbose": True,
-        }
+                "disable_certs": False,
+                "verbose": True,
+            }
+        )
     finally:
         shutil.rmtree(tmpdir)
 
 
+@pytest.fixture(scope="function")
+def nightly_task(config):
+    with open(NIGHTLY_TASK_PATH, "r") as fh:
+        return json.load(fh)
```

Comment:
Nice. Are we able to use this fixture to fix this todo? https://github.com/mozilla-releng/scriptworker-scripts/blob/55b7bbfe5a026b8ef352859a2dae9e0663ba1c75/balrogscript/tests/test_script.py#L379-L380

Reply:
Looks like yes!